packages/medusa-react/src/hooks/admin/sales-channels/mutations.ts

```diff
@@ -162,3 +163,55 @@ export const useAdminAddProductsToSalesChannel = (
     )
   )
 }
+
+/**
+ * Add a location to a sales channel
+ * @experimental This feature is under development and may change in the future.
+ * To use this feature please install the stock location in your medusa backend project.
+ * @description Add a location to a sales channel
+ * @param options
+ */
+export const useAdminAddLocationToSalesChannel = (
+  options?: UseMutationOptions<
+    Response<AdminSalesChannelsRes>,
+    Error,
+    {
+      sales_channel_id: string
+      location_id: string
+    }
+  >
+) => {
+  const { client } = useMedusa()
+  const queryClient = useQueryClient()
+  return useMutation(({ sales_channel_id, location_id }) => {
+    return client.admin.salesChannels.addLocation(sales_channel_id, {
+      location_id,
+    })
+  }, buildOptions(queryClient, [adminSalesChannelsKeys.lists(), adminSalesChannelsKeys.details(), adminStockLocationsKeys.all], options))
```

Comment:
nit: I see that some files lint have changed but here we have more than the max length, should we wrap the array? same for the next function